visual-js/.changeset/funny-shrimps-compete.md

```diff
@@ -0,0 +1,9 @@
+---
+"@saucelabs/nightwatch-sauce-visual-service": major
+"@saucelabs/visual-storybook": major
+"@saucelabs/cypress-visual-plugin": major
+"@saucelabs/wdio-sauce-visual-service": major
+"@saucelabs/visual": major
```

Comment:
Does this need to be major? Atleast in the JS ecosystem minor prerelease should effectively be a 'major' one for update checking

Reply:
We can certainly go with a minor, if that's ok with @konraddysput

Reply:
I agree with @omacranger , we should not update the major value.

Reply:
I thought we agreed on the major update due to the nature of the new diffing engine. @omacranger do you mean minor update for visual package or minor for all js packages?

Reply:
I've set all to minor as discussed.